easybuild/easyconfigs/a/AMRFinderPlus/AMRFinderPlus-4.0.23-gompi-2024a.eb

```
@@ -0,0 +1,47 @@
+easyblock = 'MakeCp'
+
+name = 'AMRFinderPlus'
+version = '4.0.23'
+
+homepage = 'https://github.com/ncbi/amr'
+description = """This software and the accompanying database are designed to find acquired antimicrobial
+ resistance genes and some point mutations in protein or assembled nucleotide sequences."""
+
+toolchain = {'name': 'gompi', 'version': '2024a'}
+
+sources = [{
+    'filename': 'amrfinderplus-%(version)s.tar.xz',
+    'git_config': {
+        'url': 'https://github.com/ncbi',
+        'repo_name': 'amr',
+        'tag': 'amrfinder_v%(version)s',
+        'recursive': True,   # <-- pull submodules so stx/ has a Makefile
+    },
+}]
+checksums = ['6a04762cb707870af569d3040cac85cc03ba881aa638f9dd34b7eb42cd4cb370']
```

```suggestion
checksums = ['72e16cefb517429716be78034371105f58f7f7e5bf601a04f2841d11de1066bc']
```

I don't think the old one is correct, perhaps it wasn't generated with eb 5.0? or old tarball from before it was made stable